build/copts.bzl

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Aurora Operations, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Extra flags we want to pass to the compilers.
+# -Wall is already set by aspect_gcc_toolchain.
+EXTRA_COPTS = [
+    "-Wextra",
+]
+
+# Since the clang toolchain we're using doesn't let us extract the default flags, we have to
+# manually specify the default flags here. These are copied from:
```

Comment:
It's too bad we need to copy, but I understand why and I think it's the best solution.  I really appreciate how you've cleanly separated the defaults from the extras we add on.  That will make it easy to check later on if the defaults have diverged.